pkg/scalers/liiklus/LiiklusService.pb.go

```diff
@@ -1,16 +1,16 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
```

Comment:
please remove this from the PR, it is not relevant and we are downgrading the protoc version